week3/api-services/security_integration.py

```
@@ -0,0 +1,359 @@
+    # Add a reusable dependency
+    async def get_current_user(request: Request, user_info = Depends(middleware.verify_api_key)):
+        return user_info
+    
+    app.get_current_user = get_current_user
+    
+    return app
+
+# Example usage functions for testing
+def create_test_api_keys():
+    """Create test API keys for development"""
+    security_client = SecurityClient()
```

Ruff (0.12.2)

Local variable `security_client` is assigned to but never used

Remove assignment to unused variable `security_client`

(F841)